packages/mask/src/plugins/NextID/components/Tip/TokenSection/index.tsx

```diff
@@ -41,6 +41,7 @@ export const TokenSection: FC = () => {
             token={token}
             amount={amount}
             maxAmount={maxAmount}
+            maxAmountDecimals={6}
```

Comment:
Some tokens have more decimals than 6.

Reply:
Of course.
In fact, I used a wrong name, it should be `maxAmountSignificant` instead.